server/config.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 	DNSSEC     string `json:"dnssec,omitempty"`
 	// Round robin A/AAAA replies. Default is true.
 	RoundRobin bool `json:"round_robin,omitempty"`
+	// round robin selection of nameservers from among those listed, rather than have all forwarded requests try the first listed server first every time.
+	NSRotate bool `json:"ns_rotate,omitempty"`
```

Comment:
R in NSRotate was capital, because it is the start of a new word in the compound word, like DnsAddr has A in capital..
I can change it if you want, but I feel it should be left the way it is..